package.nls.json

```diff
@@ -1,6 +1,7 @@
 {
     "staticWebApps.createStaticWebApp": "Create Static Web App...",
     "staticWebApps.createStaticWebAppAdvanced": "Create Static Web App... (Advanced)",
+    "staticWebApps.createStaticWebAppFromTemplate": "Create Static Web App from Template...",
```

Comment:
We probably want to change the wording from "template" to "quickstart". But need help deciding.

* "Create Static Web App with a Quickstart"
* "Static Web App Quickstart"
* "Create with Quickstart Template"
* ???